### PR TITLE
McAfee-Virus Phishing Kit 8wer3hwa

### PR DESCRIPTION
**Pull request by xVoid879**
Detects phishing pages mimicking McAfee with suspicious behavior.
title: McAfee Virus Phishing Kit 8wer3hwa
description: Detects phishing pages mimicking McAfee with suspicious behavior.
level: likely_malicous
    
references:
    # No References 

detection:

  mcafeeKeywords:
    html|contains: "mcafee"
    html|contains: "mcafee support"
    html|contains: "mcafee security"
  
  urgencyIndicators:
    html|contains: "immediate action"
    html|contains: "account locked"
    html|contains: "virus detected"

  dataExfiltration:
    html|element: input|attr: type|in: ["password", "credit-card", "social-security-number"]  
    
  realDomains:
    hostname:
      - mcafee.com

  condition: mcafeeKeywords and not realDomains and (urgencyIndicators or dataExfiltration) 

tags:
  - kit
  - target.mcafee
  - target.phishing

### CODE REVIEW

File: indicators/McAfee-Virus-8wer3hwa.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+title: McAfee Virus Phishing Kit 8wer3hwa
+description: Detects phishing pages mimicking McAfee with suspicious behavior.
+level: likely_malicous
+    
+references:
```
Comment (TheTrueSurvivor) [inline indicators/McAfee-Virus-8wer3hwa.yml:5]: Definitely needs references.

File: indicators/McAfee-Virus-8wer3hwa.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+title: McAfee Virus Phishing Kit 8wer3hwa
+description: Detects phishing pages mimicking McAfee with suspicious behavior.
```
Comment (TheTrueSurvivor) [inline indicators/McAfee-Virus-8wer3hwa.yml:2]: Needs more of a description for example,
Detects phishing pages mimicking McAfee detecting a virus on your computer using a suspicious domain checker and a keywords in the domain such as "virus detected or mcafee support. "

**Review by TheTrueSurvivor — CHANGES_REQUESTED** [review]
My thoughts on this.

File: indicators/McAfee-Virus-8wer3hwa.yml (hunk 1 above)
Comment (xVoid879) [reply indicators/McAfee-Virus-8wer3hwa.yml:5]: I don't have any references of a mcafee scam site.

File: indicators/McAfee-Virus-8wer3hwa.yml (hunk 2 above)
Comment (xVoid879) [reply indicators/McAfee-Virus-8wer3hwa.yml:2]: I will add more of description!

**Comment by IlluminatiFish** [comment]
As @TheTrueSurvivor pointed out, a rule should ideally have some references. Typically IOK rules will contain [at least two URLScan scans](https://github.com/phish-report/IOK/blob/main/CONTRIBUTING.md) as references so that the CI/CD workflows can validate that the rule works as intended. 

That being said the `dataExfiltration` search identifier in your rule will throw an error as neither `input` nor `type` are valid field names within IOK. The list of valid field names can be found listed in the [README](https://github.com/phish-report/IOK?tab=readme-ov-file#-creating-indicators) of this repository.

Additionally the `element`, `attr` and `in` field modifiers you have specified in your rule also do not exist within IOK (which uses sigma-go under the hood). A valid list of field modifiers (called comparators) can be found [here](https://github.com/bradleyjkemp/sigma-go/blob/main/evaluator/modifiers/modifiers.go#L90-L100). Keep in mind that even if a field modifier is supported by sigma-go, it may not be supported by IOK.

To better understand how to create effective IOK rules, I recommend trying your hand at the [Advent of IOK](https://phish.report/IOK/learn) and reviewing existing rules within this repository.

**Comment by xVoid879** [comment]
> As @TheTrueSurvivor pointed out, a rule should ideally have some references. Typically IOK rules will contain [at least two URLScan scans](https://github.com/phish-report/IOK/blob/main/CONTRIBUTING.md) as references so that the CI/CD workflows can validate that the rule works as intended.
> 
> That being said the `dataExfiltration` search identifier in your rule will throw an error as neither `input` nor `type` are valid field names within IOK. The list of valid field names can be found listed in the [README](https://github.com/phish-report/IOK?tab=readme-ov-file#-creating-indicators) of this repository.
> 
> Additionally the `element`, `attr` and `in` field modifiers you have specified in your rule also do not exist within IOK (which uses sigma-go under the hood). A valid list of field modifiers (called comparators) can be found [here](https://github.com/bradleyjkemp/sigma-go/blob/main/evaluator/modifiers/modifiers.go#L90-L100). Keep in mind that even if a field modifier is supported by sigma-go, it may not be supported by IOK.
> 
> To better understand how to create effective IOK rules, I recommend trying your hand at the [Advent of IOK](https://phish.report/IOK/learn) and reviewing existing rules within this repository.

**Pull request closed without merging**